src/schemas/CryptoAccountLinks.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "CryptoAccountLinks",
+  "patternProperties": {
+    "^[a-zA-Z0-9]{1,63}@[-a-zA-Z0-9]{3,16}:[-a-zA-Z0-9]{1,47}": {
+      "type": "string",
+      "pattern": "^ceramic://.+",
+      "maxLength": 1024
+    }
+  },
+  "propertyNames": {
+    "maxLength": 1024
+  },
```

Comment:
What does this do?

Reply:
Makes the schema "secure" according to ajv. Mandates that properties have limited length.

Reply:
But maxLength was also set in the patternProperty above. Why does it need to be in both places?

Reply:
One is for property name (`propertyNames` clause), another is for property value (`patternProperties/*`).

Reply:
Both have to be explicitly constrained, as both are patterns.

Reply:
Aha, I see. Thanks!